ensemble/ctf/gaussian_function_component.py

```diff
@@ -16,9 +16,7 @@
 BLACK = (0.0, 0.0, 0.0, 1.0)
 GRAY = (0.6, 0.6, 0.6, 1.0)
 POINTER_MAP = {'move': 'hand', 'resize': 'size left'}
-MIN_GAUSSIAN_STD = 2.0
-GAUSSIAN_RADIUS_STD_SCALE = 50.0
-MAX_GAUSSIAN_NUM_SAMPLES = 256
+MAX_NUM_SAMPLES = 256
 GAUSSIAN_MINIMUM_RADIUS = 0.03
```

Comment:
It would be nice to have a smaller minimum radius. If the radius is too small I get weird behavior. Making this `0.005` seems to work fine